assets/styles/style.css

```diff
@@ -1024,3 +1024,92 @@ div.header-user img.avatar {
 		z-index: 100;
 	}
 }
+.directory.members ul#members-list {
```

Comment:
It might be nice to add a little comment bar above this new stuff to mark it as a "member directory" section or something to help keep things organized.

Reply:
Done.